pkg/scan/scan.go

```diff
@@ -179,7 +188,7 @@ func handleManifest(reg types.Registry, manifestType, remote, ref string) (diges
 		}
 		layers := parseLayers(manifest.Layers)
 		return dig, layers, nil
-	case ociSpec.MediaTypeImageManifest:
+	case registry.MediaTypeImageManifest:
```

Comment:
Thoughts on using the types from [opencontainers/image-spec](https://github.com/opencontainers/image-spec/blob/48fd0740a26631ae0f59f514cbf017bcf841d00b/specs-go/v1/mediatype.go#L24-L28) for `MediaTypeImageManifest` and `MediaTypeImageIndex`?